### DISCUSSION TITLE
With TrafficSplit no more in the dashboard, how to check the weight?

### DISCUSSION THREAD

**Discussion by n3wt0n** [linkerd/linkerd2#11015] [Help] [answered]
With Linkerd removing the SMI dependencies, and having that into linkerd-smi, the `Traffic Splits` page has been removed from the portal (linkerd-viz).

We were relying on that because we do practice canary rollout (with ArgoCD Rollouts), and the deployment may take up to 1 hour.

Without that page, what is the recommended way to check for the weight associated with the different deployments (canary vs stable, etc) during a rollout?

**Comment by alpeb** [comment, marked as answer]
You still can see the service weights in a TrafficSplit in the dashboard, in the Outbound section for the apex Service:

![image](https://github.com/linkerd/linkerd2/assets/554287/13fc0c46-f04d-4623-b1ae-074f43c13490)

**Reply by n3wt0n** [reply]
Thanks!
